src/components/NamespaceSelector/NamespaceSelectorTs.ts

```diff
@@ -49,12 +49,26 @@ export class NamespaceSelectorTs extends Vue {
    */
   @Prop({ default: null }) value: string
 
+  /**
+   * isLink
+   * True represents namespaces link, or false represents creating subnamespaces
+   * @type {string}
+   */
+  @Prop({ default: false }) isLink: boolean
+
   /**
    * Namespaces names
    * @type {[h: string]: string}
    */
   public namespaces: NamespaceModel[]
 
+  /**
+   * filter Namespaces
+   */
+  get getNamespaces() {
```

Comment:
rename to `get linkableNamespaces` and should always return only **namespaces that are not linked**